library/alloc/src/string.rs

```diff
@@ -3151,7 +3152,8 @@ impl Iterator for IntoChars {
             None => None,
             Some((_, ch)) => {
                 let offset = iter.offset();
-                drop(self.bytes.drain(..offset));
+                // SAFETY: `offset` is a valid index.
```

Comment:
@programmerjake Updated.

Not sure if it's better to use `.expect` here. I don't find a similar pattern yet.

Reply:
`advance_by` should never return `Err` here, so ignoring it seems fine enough.

Reply:
This isn't an unsafe block so this shouldn't be a SAFETY comment. A normal comment is sufficient.

Reply:
@Amanieu  Ok. Updated.